docs/src/technical-considerations.md

```diff
@@ -0,0 +1,19 @@
+# Technical Considerations
+
+## Optimizing Tile Requests in titiler-stacapi
+
+To enhance the efficiency of tile requests in titiler-stacapi, it's essential to set appropriate zoom levels and bounding boxes and be mindful of the number of requests made to the STAC API. This guide will walk you through understanding and applying these settings to improve performance and avoid overloading the service with expansive queries.
+
+### Understanding Zoom Levels
+
+Zoom levels dictate the level of detail in the tiles requested. Specifying a range (min/max) can significantly optimize request times and resource usage.
+
+### Bounding Box Considerations
+
+The bounding box limits the geographical area of interest, preventing unnecessarily broad tile generation. Define this based on your application's specific needs to reduce load times and server strain.
```

Comment:
In another PR it would be nice to add a Notebook showing how we can get those informations

Reply:
ah! that's what you meant here: https://github.com/developmentseed/titiler-stacapi/issues/2